libtransact/src/context/manager/mod.rs

```diff
@@ -60,6 +61,10 @@ impl ContextLifecycle for ContextManager {
             .build()?;
         Ok(new_transaction_receipt)
     }
+
+    fn clone_box(&self) -> Box<dyn ContextLifecycle> {
```

Comment:
This feels a bit wrong with this implementation, as it copies all the content (and therefore doesn't properly operate on any new contexts created in the original object.

Reply:
Though, perhaps it creates a bit more expediency on getting #6 correct and merged.